hack/jenkins/e2e.sh

```diff
@@ -124,6 +124,21 @@ GCE_PARALLEL_FLAKY_TESTS=(
     "Services.*release.*load\sbalancer"
     )
 
+GCE_SOAK_CONTINUOUS_SKIP=(
```

Comment:
I have no idea how this list was generated - I copied it from Jenkins setup. I don't know who or how created it.